test/unit3/test_math.py

```diff
@@ -1481,6 +1481,14 @@ def testIsinf(self):
         self.assertFalse(math.isinf(2**1024))
         self.assertFalse(math.isinf(2**2048))
 
+    def testLcm(self):
+        self.assertEqual(math.lcm(), 1)
+        self.assertEqual(math.lcm(2), 2)
+        self.assertEqual(math.lcm(0), 0)
+        self.assertEqual(math.lcm(576559230871654959816130551884856912003141446781646602790216406874), 576559230871654959816130551884856912003141446781646602790216406874)
+        self.assertEqual(math.lcm(1, 2, 0, 5, 6), 0)
+        self.assertEqual(math.lcm(2, 3, 4, 5, 6), 60)
```

Comment:
It would be good to see the cpython lcm tests here
(commenting out the ones that don't work)
https://github.com/python/cpython/blob/main/Lib/test/test_math.py#L1051-L1093

If you end up touching `gcd` likewise:
https://github.com/python/cpython/blob/main/Lib/test/test_math.py#L705-L745

Reply:
Thanks, thought of that but in the end I forgot to add it)